src/main/java/com/linecorp/armeria/client/Clients.java

```diff
@@ -40,6 +43,28 @@
     }
 
     /**
+     * Creates a new client that connects to the static server group specified by {@code serverGroup}
+     * using the default {@link RemoteInvokerFactory}.For example,
+     * <p>
+     * <pre>{@code
+     * StaticServerGroup serverGroup = new StaticServerGroup();
+     * serverGroup.addHost("127.0.0.1", 1234, 2).addHost("127.0.0.1", 2345, 4)
+     *     .strategy(StaticServerGroup.SelectStrategyType.ROUND_ROBIN).registry("myGroup");
+     *
+     * Clients.newClient("tbinary+http://myGroup/...", ...,"myGroup",...);
```

Comment:
Could you explain why we need to specify `serverGroup`? Perhaps we could check if the hostname in the URI is the name of a `ServerGroup` to pick it up automatically?